lib/src/widgets/html_editor_widget_web.dart

```diff
@@ -497,6 +497,19 @@ class _HtmlEditorWidgetWebState extends State<HtmlEditorWidget> {
                   event.preventDefault();
                   window.parent.postMessage(JSON.stringify({"view": "$createdViewId", "type": "toDart: onDragLeave", "types": event.dataTransfer.types}), "*");
                 });
+
+                document.getElementsByClassName('note-editor')[0].addEventListener("dragover", function(event) {
+                  if (event.dataTransfer.types.includes("Files")) {
+                    event.preventDefault();
+                    window.parent.postMessage(JSON.stringify({"view": "$createdViewId", "type": "toDart: onDragEnter", "types": event.dataTransfer.types}), "*");
```

Comment:
Plese use name other function avoid side effect :

```suggestion
                    window.parent.postMessage(JSON.stringify({"view": "$createdViewId", "type": "toDart: onDragover", "types": event.dataTransfer.types}), "*");
```

Reply:
Done